mypy_stubs/waffle/defaults.pyi

```diff
@@ -0,0 +1,28 @@
+from _typeshed import Incomplete
+
+COOKIE: str
+TEST_COOKIE: str
+SECURE: bool
+MAX_AGE: int
+CACHE_PREFIX: str
+CACHE_NAME: str
+FLAG_CACHE_KEY: str
+FLAG_USERS_CACHE_KEY: str
+FLAG_GROUPS_CACHE_KEY: str
+ALL_FLAGS_CACHE_KEY: str
+SAMPLE_CACHE_KEY: str
+ALL_SAMPLES_CACHE_KEY: str
+SWITCH_CACHE_KEY: str
+ALL_SWITCHES_CACHE_KEY: str
+FLAG_DEFAULT: bool
+SAMPLE_DEFAULT: bool
+SWITCH_DEFAULT: bool
+READ_FROM_WRITE_DB: bool
+CREATE_MISSING_FLAGS: bool
+CREATE_MISSING_SAMPLES: bool
+CREATE_MISSING_SWITCHES: bool
+LOG_MISSING_FLAGS: Incomplete
+LOG_MISSING_SAMPLES: Incomplete
+LOG_MISSING_SWITCHES: Incomplete
+OVERRIDE: bool
+ENABLE_ADMIN_PAGES: bool
```

Comment:
question (non-blocking): when you create stubs, do you keep everything that `mypy` creates for the module, even if it's not used in our code?

Reply:
In this case, I generated but didn't alter `defaults.pyi`. Others like `testutils.pyi` I did clean up. I'm not sure what the policy should be! I think I justified my laziness in that these would be removed next release, but who knows when that will be, `django-waffle` might be abandoned...